network/alsp/manager/manager.go

```diff
@@ -314,16 +311,18 @@ func (m *MisbehaviorReportManager) onHeartbeat() error {
 
 			// TODO: this can be done in batch but at this stage let's send individual notifications.
 			//       (it requires enabling the batch mode end-to-end including the cache in middleware).
-			// as long as record.Penalty is NOT below model.DisallowListingThreshold,
+			// as long as record.Penalty is NOT below model.disallowListingThreshold,
```

Comment:
```suggestion
			// as long as record.Penalty is NOT below model.DisallowListingThreshold,
```

Reply:
Thanks for the catch! Fixed e5cf8fd949cbc7cd2e5ab0b07dcf72a96cf03686